code/modules/jobs/job_mail.dm

```diff
@@ -506,12 +518,18 @@
 		/obj/item/assembly/flash/handheld = 7,
 		/obj/effect/spawner/mail/rdonut = 5,
 		//we don't have boomerangs here, ask ausstation
-		/obj/item/melee/classic_baton/police/telescopic = 1,
+		/obj/item/melee/classic_baton/police/telescopic = 2,
 
 		//NSV13
 		/obj/item/reagent_containers/food/drinks/drinkingglass/quadruple_sec = 8,
 		/obj/item/toy/figure/secofficer = 5,
 		/obj/item/reagent_containers/food/drinks/drinkingglass/quintuple_sec = 4,
+		// They're mocking you boy
+		/obj/item/clothing/under/syndicate/tacticool = 10,
+		// he will buy the SWAT pack anyway
+		/obj/item/combatgloves = 2,
+		// CHRISTMAS!?
+		/obj/item/gun/ballistic/shotgun/lever_action = 1,
```

Comment:
This is considered a hazard goodie, so please add this to the hazard goodie list in mail.dm so an admin log will be sent whenever it spawn.